index.html

```diff
@@ -2,17 +2,20 @@
 <html class="page" lang="ru">
 
 <head>
-  <meta charset="utf-8">
+  <meta charset="utf-8" />
   <title>HTML Academy: Нёрдс</title>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="css/normolize.css" />
+  <link rel="stylesheet" href="css/style.css" />
 </head>
 
 <body class="page-body">
   <header class="main-header">
     <nav class="main-navigation">
-      <a>
-        <img src="img/index-logo.svg" width="156" height="65" alt="Logo">
+      <a class="main-header-logo">
+        <img src="img/index-logo.svg" width="156" height="65" alt="Logo" />
```

Comment:
```suggestion
        <img src="img/index-logo.svg" width="156" height="65" alt="Logo">
```
В современном синтаксисе HTML слэш не ставится в конце непарного тега.